config/model/mobilenetv3/mobilenetv3-small-classification.yaml

```diff
@@ -0,0 +1,28 @@
+model:
+  task: classification
+  checkpoint: ./weights/mobilenetv3/mobilenet_v3_small.pth
+  fx_model_checkpoint: ~
+  resume_optimizer_checkpoint: ~
+  freeze_backbone: False
+  architecture:
+    full: ~ # auto
+    backbone:
+      name: mobilenetv3_small
+      block_info: # [in_channels, kernel, expended_channels, out_channels, use_se, activation, stride, dilation]
```

Comment:
`block_info` format should be equalized with other models in later.

Reply:
Yeah. We should discuss about it.